.github/workflows/march-image-build-push.yml

```diff
@@ -0,0 +1,78 @@
+name: 'multi-arch image build'
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+      - 'release-*'
+    tags:
+      - 'v*'
+  schedule:
+    - cron: '0 0 * * *'
+
+env:
+  IMAGE_NAME: tackle2-hub
+  IMAGE_REGISTRY: quay.io
+  IMAGE_NAMESPACE: konveyor
+
+jobs:
+  push-quay:
+    name: Build and Push Manifest
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    steps:
+    - name: Checkout Push to Registry action
+      uses: actions/checkout@main
+
+    - name: Install latest podman
+      if: matrix.install_latest
+      run: |
+        bash .github/install_latest_podman.sh
+
+# This package is not new enough for ppc64le builds
```

Comment:
Should we remove this if we can use the docker action?

Reply:
Yes, removed! It works fine with the docker action alternative.